### DISCUSSION TITLE
Cypress and SSR (Server Side Rendering)

### DISCUSSION THREAD

**Discussion by matheusbarcelos** [cypress-io/cypress#26595] [General]
I'm developing tests with Cypress to validate an SSR (Server Side Rendering) application
I have some problems running the tests, some elements are available on the page. But they are being rendered on the server.
This way Cypress interacts with the element, but the application does not respond. Example: clicks but does not take to another page.

I looked everywhere, I even found this example from Bahmutov. But I haven't found a way for Cypress to be able to 'wait' for the components to render.

The following article:
https://glebbahmutov.com/blog/ssr-e2e/

Kindly, has anyone worked with Cypress in SSR applications?
Could you give me a tip, or where can I get more information?

**Comment by nmcivor501** [comment]
@matheusbarcelos Were you able to find anything else out about Cypress SSR ?
